src/test/java/net/openhft/chronicle/map/MissSizedMapsTest.java

```diff
@@ -29,20 +32,20 @@ public void testSmallEntries() throws IOException, URISyntaxException {
     }
 
     private void check(final ChronicleMap<String, String> actual) throws IOException, URISyntaxException {
-        URI uri = MissSizedMapsTest.class.getResource("/input.txt").toURI();
+        URI uri = MissSizedMapsTest.class.getResource("/input.txt.gz").toURI();
 
-        List<String> strings = Files.readAllLines(Paths.get(uri));
         Map<String, String> expected = new HashMap<>();
 
         int maxKey = 0;
         int maxValue = 0;
-        for (String s : strings) {
-            String[] split = s.split("&");
-            expected.put(split[0], split[1]);
-            actual.put(split[0], split[1]);
-            maxKey = Integer.max(maxKey, split[0].length());
-            maxValue = Integer.max(maxValue, split[1].length());
-
+        try (BufferedReader br = new BufferedReader(new InputStreamReader(new GZIPInputStream(uri.toURL().openStream())))) {
+            for (String s; (s = br.readLine()) != null; ) {
+                String[] split = s.split("&");
+                expected.put(split[0], split[1]);
+                actual.put(split[0], split[1]);
+                maxKey = Integer.max(maxKey, split[0].length());
```

Comment:
could be a static import

Reply:
could be != should be

static imports are a matter of taste, and quite often obscure code intention